tests/client/amo/components/TestFooter.js

```diff
@@ -36,7 +36,10 @@ describe('Footer', () => {
       'Legal notices');
     assert.equal(root.querySelector('.Footer-legal').href,
       'https://www.mozilla.org/en-GB/about/legal/');
+    // This link isn't localized because MDN will 404 on some
+    // locales and not others.
```

Comment:
Worth including the link to the original issue in this code base too, if you want.